main.cpp

```diff
@@ -20,67 +20,96 @@ You are going to write 10 UDTs in project3.
-Sub Object 1: Steering Wheel
-    Name 4 things you'll find on the:    Steering Wheel
-        1) paddle shifters
-        2) 'cruise control' controls
-        3) horn
-        4) volume control
-    Name 2 things you can do with the:   Steering Wheel
-        1) adjust cruise control settings.
-        2) steer
-        
-Sub Object 2: Instrument Cluster
-    Name 4 things you'll find on the:   Instrument Cluster
-        1)  speedometer
-        2)  miles per gallon rating
-        3)  gas meter
-        4)  temperature meter
-    Name 3 things you can do with the:   Instrument Cluster
-        1)  display speed
-        2)  display acceleration
-        3)  display temperature
-    
-Sub Object 3: Environment Controls
-    Name 3 things you'll find on the:    Environment Controls
-        1)  AC on/off button
-        2)  Auto AC button
-        3)  temperature adjust switch
-    Name 3 things you can do with the:   Environment Controls
-        1)  turn AC on
-        2)  engage Auto AC
-        3)  adjust temperature
-
-Sub Object 4: Infotainment System
-    Name 3 things you'll find on the:    Infotainment System
-        1)  song/radio selection
-        2)  volume adjustment icon
-        3)  play/pause icon
-    Name 3 things you can do with the:   Infotainment System
-        1)  select song/station
-        2)  adjust volume
-        3)  play/pause
-
-Sub Object 5: Seat 
-    Name 3 things you'll find on the:    Seat
-        1)  seat belt
-        2)  position adjust levers
-        3)  head cushion
-    Name 2 things you can do with the:   Seat
-        1)  adjust seat settings
-        2)  put on/off seatbelt
+1)  Fill in #1 - 4 with a random UDT in plain english
+    These 4 UDTs do not need to be related.
+        a) For each plain-english UDT, write out 5 traits or properties and 3 things it can do
+        b) remember: these properties will eventually become primitives
 */
+
+/*
+ example:  
+ 
+Thing: Car Wash   
+    5 properties:
+        - number of vacuum cleaners
+        - number of eco-friendly cleaning supplies
+        - stores the amount of water used per week.
+        - stores amount of profit made per week
+        - number of cars serviced per day
+    3 things it can do:
+        - wash and wax car
+        - charge customer
+        - detail the car interior
+
+    Notice that I did not use "has a vacuum cleaner" or "Has eco-friendly cleaning supplies" as one of the properties
+    
+    Writing 'has a ___" checks whether or not your object **has the ability to do something**.
+    Instead, I wrote "number of vacuum cleaners" and "number of eco-friendly cleaning supplies".  
+    
+    These are specific objects or amounts. 
+    
+    In C++ terms, this means to I want you to avoid using 'bool' (has a) as a member variable type.
+    Instead, prefer the other primitive types.
+ */
+
+/*
+Thing 1) Prophet-6
+5 properties:
+    1)  oscillators
+    2)  filters
+    3)  envelopes
+    4)  VCAs
+    5)  Keyboard
```

Comment:
These are objects, not primitives!   You need to pick things that can be represented with `int float double bool char std::string`
- [ ] revise